app/models/user.rb

```diff
@@ -0,0 +1,12 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :likes
+  has_many :comments
+  attribute :posts_counter, :integer, default: 0
+  validates :name, presence: true
+  validates :posts_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+
+  def recent_posts
+    posts.order(created_at: :desc).limit(3)
+  end
+end
```

Comment:
- [ ] Great job so far @ctokoli There are a few changes I will suggest you implement and you'll be on your way to the next Milestone :wink: 
- [ ] Great job adding most of the tests, however, I noticed that per the project requirements, you were asked to `Add unit specs for all of your models' methods and validations.` But currently, this `recent_posts` method in your `User Model` hasn't been tested. So kindly include tests for that method as well :wink: